dspace-api/src/main/java/org/dspace/app/bulkedit/DSpaceCSV.java

```diff
@@ -141,18 +138,18 @@ public DSpaceCSV(boolean exportAll) {
     /**
      * Create a new instance, reading the lines in from file
      *
-     * @param f The file to read from
+     * @param inputStream the inputstream to read from
      * @param c The DSpace Context
      * @throws Exception thrown if there is an error reading or processing the file
      */
-    public DSpaceCSV(File f, Context c) throws Exception {
+    public DSpaceCSV(InputStream inputStream, Context c) throws Exception {
         // Initialise the class
         init();
 
         // Open the CSV file
         BufferedReader input = null;
         try {
-            input = new BufferedReader(new InputStreamReader(new FileInputStream(f), "UTF-8"));
+            input = new BufferedReader(new InputStreamReader(inputStream, "UTF-8"));
```

Comment:
Once you update the import to be StandardCharsets, this line should likely replace `"UTF-8"` with `StandardCharsets.UTF_8` for consistency.